tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -130,9 +128,25 @@ host: '<host>', port: 4000,user: '<username>', password: '<your_password>', data
 </div>
 </SimpleTab>
 
-## Where is the CA root path on my system?
+## Root certificate management
+
+### Root certificate issuance and validity
+
+TiDB Serverless Tier uses certificates from [Let's Encrypt](https://letsencrypt.org/) as a Certificate Authority (CA) for TLS connection between clients and TiDB Serverless Tier clusters.If the Serverless Tier certificate expires, it will be automatically rotated without affecting the normal operation of your dedicated tier cluster and the established TLS secure connection.
```

Comment:
```suggestion
TiDB Serverless Tier uses certificates from [Let's Encrypt](https://letsencrypt.org/) as a Certificate Authority (CA) for TLS connection between clients and TiDB Serverless Tier clusters.  Once the Serverless Tier certificate expires, it will be automatically rotated without affecting the normal operations of your cluster and the established TLS secure connection.
```